docs/core/getting-started/install/win-vs2015.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Install for Windows - Visual Studio 2015
+description: Install for Windows - Visual Studio 2015
+keywords: .NET, .NET Core, Visual Studio 2015, VS2015, C#, Getting Started, Acquisition, Cross Platform
+author: kendrahavens
+ms.author: kendrahavens
+ms.date: 12/19/2016
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 
+---
+# Install for Windows - Visual Studio 2015
+## 1. Download Visual Studio 2015
+- Make sure you have Visual Studio 2015 Update 3 installed.
+  ## [Download Visual Studio 2015 with Update 3](https://go.microsoft.com/fwlink/?LinkId=691978)
+- Or just download [Visual Studio 2015 Update 3](https://go.microsoft.com/fwlink/?LinkId=691129) if you already have Visual Studio 2015 installed.
```

Comment:
Our windows prerequisites page calls out Update 3.3 as well, take a look: https://docs.microsoft.com/en-us/dotnet/articles/core/windows-prerequisites

also, what are your plans for that particular topic? it also has some notes about the offline installer option that should be part of both VS topics (I know the VS 2015 one is correct; I'm not sure the status of offline installer for VS 2017).